pkg/cmd/pulumi/stack_init_test.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/pulumi/pulumi/pkg/v3/backend"
+	"github.com/stretchr/testify/assert"
+)
+
+// This test demonstrates that validateCreateStack will filter
+// out teams consisting exclusively of whitespace. NB: It's not intended
+// to fully validate the correctness of team names. For example, it doesn't
+// check for illegal punctuation, length, or other measures of correctness.
+// To keep the codebase DRY, we pass along team names as-is to the Service,
+// with the exception of trimming whitespace, and allow the Service to
+// validate them.
+func TestValidateCreateStackOpts(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct {
+		name                 string
+		rawTeams, validTeams []string
```

Comment:
nit: valid/invalid terminology is off here. There are blank and non-blank cases.
If they were "invalid", we'd expect an error.
Maybe we should name these "giveTeams" and "wantTeams"?
The list of `--teams` values we passed in, and those we expect back?

Reply:
I changed the names to `inputTeams` and `expectedTeams`. I can do `giveTeams` and `wantTeams` if you'd prefer.

Aside: In side projects, my preference is to structure as many tests as possible with `expected` and `observed terminology. Each test should have somewhere that checks `expected == observed`. I think of it kind of like a scientific hypothesis that you're attempting to validate: write your expectations up front, and compare your observed values against them. :)